pkg/util/crd_defaults.go

```diff
@@ -28,6 +28,17 @@ import (
 	//samplev1 "github.com/blackducksoftware/synopsys-operator/pkg/api/sample/v1"
 )
 
+const (
+	// AlertCRDName is the name of an Alert CRD
+	AlertCRDName = "alerts.synopsys.com"
```

Comment:
These should be singular (alert.synopsys.com)

Reply:
nope. All the CRD names are plural